qutebrowser/misc/sessions.py

```diff
@@ -214,10 +214,18 @@ def _save_tab(self, tab, active):
             data['history'].append(item_data)
         return data
 
-    def _save_all(self):
+    def _save_all(self, only_active_window = False):
```

Comment:
This should probably be a keyword-only argument (here and elsewhere), i.e. `self, *, arg=False`. Also, no spaces around `=` for argument default values.